test/e2e/messaging.test.ts

```diff
@@ -0,0 +1,170 @@
+import { expect } from 'chai'
+import request from 'supertest'
+import { describe } from 'mocha'
+import { KeyType, ConnectionRecord } from '@aries-framework/core'
+
+import { getConfig } from './fixtures/config.js'
+
+describe('Messaging', async function () {
+  const config = getConfig()
+  const aliceClient = request(config.alice.portalUrl)
+  const bobClient = request(config.bob.portalUrl)
+  const aliceVeritableClient = request(config.alice.veritableUrl)
+  const bobVeritableCLient = request(config.bob.veritableUrl)
+  let bobTestDid: string | undefined = undefined
+  let aliceTestDid: string | undefined = undefined
+
+  before(async function () {
+    // import dids on Alice and Bob
+    await request(config.alice.veritableUrl)
+      .post('/dids/import')
+      .send({
+        did: config.alice.did,
+        privateKeys: [
+          {
+            keyType: KeyType.Ed25519,
+            privateKey: config.alice.privateKey,
+          },
+        ],
+        overwrite: true,
+      })
+      .set('Accept', 'application/json')
+      .expect(200)
+
+    await request(config.bob.veritableUrl)
+      .post('/dids/import')
+      .send({
+        did: config.bob.did,
+        privateKeys: [
+          {
+            keyType: KeyType.Ed25519,
+            privateKey: config.bob.privateKey,
+          },
+        ],
+        overwrite: true,
+      })
+      .set('Accept', 'application/json')
+      .expect(200)
+  })
+
+  after(async function () {
+    // clear all connections
+    const { body: aliceConns } = await aliceVeritableClient.get(`/connections`)
+    for (const { id } of aliceConns) {
+      await aliceVeritableClient.delete(`/connections/${id}`)
+    }
+    const { body: bobConns } = await bobVeritableCLient.get(`/connections`)
+    for (const { id } of bobConns) {
+      await bobVeritableCLient.delete(`/connections/${id}`)
+    }
```

Comment:
maybe this can go in some shared file as well? I haven't checked if it's exactly the same as in `connection.test.ts`